markup.js

```diff
@@ -267,4 +201,39 @@ function buildKeyboard (buttons, options) {
   return result
 }
 
+function openTag (tag, params) {
+  return params
+    ? `<${tag} ${Object.entries(params)
+        .map(param => `${param[0]}="${param[1]}"`)
+        .join(' ')}>`
+    : `<${tag}>`
+}
+
+function closeTag (tag) {
+  return `</${tag}>`
```

Comment:
```suggestion
  return `</${tag || ''}>`
```
Maybe this?

Reply:
I think this is not necessary, the tag will always not be empty. In addition, everyone has a closing pair.